app/helpers/webpages_helper.rb

```diff
@@ -8,6 +8,13 @@ def display_image(model, homepage = false)
       ((image_pack_tag "default-book-cover-index.png") if homepage.present?)
   end
 
+  def hot_cover(book)
```

Comment:
The spec for this method is still in the books_helper_spec.  Please move that to the webpages_helper_spec.

Reply:
done